baseplate/lib/message_queue.py

```diff
@@ -140,6 +188,11 @@ def queue_tool() -> None:
         help="if creating the queue, what to set the maximum message size to",
     )
     parser.add_argument("queue_name", help="the name of the queue to consume")
+    parser.add_argument(
```

Comment:
A better choice might be to make this an option `--queue-type` with values `posix` or `memory` where the default value is `posix`. This would enable us to cleanly add another type in the future if we want to.

Reply:
I agree, you can constrain the options (and provide useful CLI help hints) by using a defined set of `[choices](https://docs.python.org/3/library/argparse.html#choices)`